components/quick_input/quick_input.tsx

```diff
@@ -11,65 +11,75 @@ import OverlayTrigger from 'components/overlay_trigger';
 import AutosizeTextarea from 'components/autosize_textarea';
 import Constants from 'utils/constants.jsx';
 
+export type Props = {
+
+    /**
+     * Whether to delay updating the value of the textbox from props. Should only be used
+     * on textboxes that to properly compose CJK characters as the user types.
+     */
+    delayInputUpdate?: boolean;
+
+    /**
+     * An optional React component that will be used instead of an HTML input when rendering
+     */
+    inputComponent?: ReactComponentLike;
+
+    /**
+     * The string value displayed in this input
+     */
+    value: string;
+
+    /**
+     * When true, and an onClear callback is defined, show an X on the input field that clears
+     * the input when clicked.
+     */
+    clearable?: boolean;
+
+    /**
+     * The optional tooltip text to display on the X shown when clearable. Pass a components
+     * such as FormattedMessage to localize.
+     */
+    clearableTooltipText?: string | ReactElementLike;
+
+    /**
+     * Callback to clear the input value, and used in tandem with the clearable prop above.
+     */
+    onClear?: () => any;
```

Comment:
If `onClear` doesn't return anything, the return should be `void`
```suggestion
    onClear?: () => void;
```

Reply:
Obviously! Fixed in b29e36ef80f1